pkgs/top-level/python-packages.nix

```diff
@@ -32131,6 +32131,128 @@ EOF
 
   treq = callPackage ../development/python-modules/treq { };
 
+  nss = buildPythonPackage rec {
```

Comment:
could you put this new package, as well as all other new Python packages, in a separate file in `pkgs/development/python-modules/<modulename>/default.nix`.